### PR TITLE
the botany chem fixes+

### PR DESCRIPTION
**Pull request by Baldir-Odensen**
well posibly should atmoize this

## About The Pull Request
some plants produces non existant chemicals. also adds sodium to ghost chilli because zeta said it would fit in it.

## Why It's Good For The Game
well plants should produce stuff. and sodium production is funni

## Changelog
:cl:
tweak: added sodium to ghost chilli
fix: made it so toxin healing plants produce toxin healing.
/:cl:

### CODE REVIEW

File: code/modules/hydroponics/grown/kudzu.dm [hunk 1]
```diff
@@ -14,7 +14,7 @@
 	growthstages = 4
 	rarity = 30
 	var/list/mutations = list()
-	reagents_add = list(/datum/reagent/medicine/C2/multiver = 0.04, /datum/reagent/consumable/nutriment = 0.02)
+	reagents_add = list(/datum/reagent/medicine/antitoxin = 0.04, /datum/reagent/consumable/nutriment = 0.02)
```
Comment (triplezeta) [inline code/modules/hydroponics/grown/kudzu.dm:17]: make this charcoal instead, as that is what multiver was originally

File: code/modules/hydroponics/grown/mushrooms.dm [hunk 2]
```diff
@@ -21,7 +21,7 @@
 	growthstages = 4
 	genes = list(/datum/plant_gene/trait/plant_type/fungal_metabolism)
 	growing_icon = 'icons/obj/hydroponics/growing_mushrooms.dmi'
-	reagents_add = list(/datum/reagent/medicine/morphine = 0.35, /datum/reagent/medicine/C2/multiver = 0.35, /datum/reagent/consumable/nutriment = 0)
+	reagents_add = list(/datum/reagent/medicine/morphine = 0.35, /datum/reagent/medicine/antitoxin = 0.35, /datum/reagent/consumable/nutriment = 0)
```
Comment (triplezeta) [inline code/modules/hydroponics/grown/mushrooms.dm:24]: this too

**Review by triplezeta — CHANGES_REQUESTED** [review]

File: code/modules/hydroponics/grown/kudzu.dm (hunk 1 above)
Comment (Baldir-Odensen) [reply code/modules/hydroponics/grown/kudzu.dm:17]: Having charcoal as a natural product of a plant growing doesn't make much sense to me. Nor is it intuitive.

**Review by triplezeta — APPROVED** [review]
cool

**Review by MarkSuckerberg — APPROVED** [review]

**Review by Superyodeler — APPROVED** [review]

**Pull request merged**